submissions/BlueLamantine/html-css-popup/style.css

```diff
@@ -0,0 +1,145 @@
+:root {
+  --bg-grey: #333333ad;
+  --lightorange: #ffd047;
+}
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body,
+html {
+  height: 100%;
+}
+body {
+  font-weight: bold;
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+  color: black;
+}
+ul {
+  display: flex;
+}
+ul li {
+  list-style: none;
+}
+.topnav {
+  overflow: hidden;
+  background-color: var(--bg-grey);
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+.topnav a,
+.topnav label {
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 20px;
+}
+a:hover,
+.topnav label:hover,
+a:focus,
+.topnav input:focus + label {
+  background-color: var(--lightorange);
+  color: black;
+  outline: none;
+}
+.nav-item{
+  float: left;
+}
+#menu-toggle,
```

Comment:
Avoid styling by `#id` and HTML tag, separate responsibilities:
- HTML tag for markup;
- CSS class for styling;
- id for JS logic, linking label with input

Reply:
It's a fair point, no more stylized by ```#id```, fixed it.

Reply:
My point is if you can avoid styling by `#id` avoid it)